.editorconfig

```diff
@@ -17,4 +17,8 @@ insert_final_newline = true
 trim_trailing_whitespace = true
 indent_style = space
 indent_size = 4
-max_line_length = 80
\ No newline at end of file
+max_line_length = 80
+
+# TS/JS-Files
+[*.{ts,js}]
+indent_size = 2
```

Comment:
Is there a reason, why TypoScript and JavaScript should have an indent of 2 and PHP and other code an indent of 4?
Can we treat this in the same way and have an indent of 4.

Reply:
It's just the TYPO3 standard (see links). So I thought that's the way to go for a TYPO3 extension.

- https://github.com/TYPO3/typo3/blob/main/Build/.eslintrc.js#L32
- https://github.com/TYPO3/typo3/blob/8fd523045b5f7f81cd4d0c6642f4ed0e732dbbdd/.editorconfig#L15-L17

Reply:
OK. Can we extract

```
[*]
charset = utf-8
end_of_line = lf
indent_style = space
indent_size = 4
insert_final_newline = true
trim_trailing_whitespace = true
```

too. Because there are some rules arn't defined for js.

Reply:
Done.